bench/bench_server.ml

```diff
@@ -0,0 +1,104 @@
+(* A multi-domain server handles HTTP-like requests from many clients running across multiple domains. *)
+
+open Eio.Std
+
+let n_client_domains = 4
+let n_server_domains = 4
+let n_connections_per_domain = 25
+let n_requests_per_connection = 1000
```

Comment:
Perhaps get these from environment variables? I can crank these up on some of my 128 core machines :-)

Reply:
OK, I've updated it so you can pass these as arguments.